### PR TITLE
versionize-docs.sh takes release branch as arg.

### PR DESCRIPTION
**Pull request by david-mcmahon**

### CODE REVIEW

File: anago [hunk 1]
```diff
@@ -364,10 +364,10 @@ prepare_tree () {
   # the NON-master branch itself, versionize the docs
   if [[ "$PARENT_BRANCH" == master && "$current_branch" != "master" ]]; then
     logecho -n "Versionizing docs for ${RELEASE_VERSION[$label]}: "
-    logrun -s $TREE_ROOT/build/versionize-docs.sh ${RELEASE_VERSION[$label]}
+    logrun -s $TREE_ROOT/build/versionize-docs.sh $current_branch
```
Comment (lavalamp) [inline anago]: I thought style was to use "${var}" for variable references.

Comment (david-mcmahon) [inline anago]: It is in code that already uses that style.  Thankfully in this repo I choose not to use that silliness.

**Comment by lavalamp** [comment]
lgtm

**Comment by david-mcmahon** [comment]
ref https://github.com/kubernetes/kubernetes/issues/27205

**Pull request merged**